databrowser/src/components/pill/PillLinkGroupItem.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    :class="{
+      'text-green-500 bg-opacity-10 bg-green-500 border-green-500': isSelected,
+      'hover:bg-gray-300': !isSelected,
+      'rounded-l-full border-l': isFirst,
+      'rounded-r-full border-r': isLast,
+    }"
+    class="
+      overflow-hidden
+      last:pr-1
+      first:pl-1
+      border-t border-b border-gray-500
+    "
+  >
+    <router-link
+      :class="{
+        'border-green-500': isSelected,
+        'border-l': !isFirst,
+        'border-r': !isLast,
+      }"
+      :to="item.url"
+      class="block border-transparent"
+      role="button"
+      tabindex="0"
+    >
+      <span class="block py-1 px-2 font-semibold">{{ item.label }}</span>
+    </router-link>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from '@vue/runtime-core';
+import { computed } from 'vue';
+import { useRoute } from 'vue-router';
+
+export type PillLinkGroupType = {
```

Comment:
In the code we did not suffix types with the word `Type`. That's why I would suggest to use another word here. For example: `PillLinkGroupItem`.